dataland-frontend/tests/e2e/specs/low/general/BackButton.ts

```diff
@@ -2,6 +2,8 @@ describe('Back Button test suite', () => {
     it('company eu taxonomy page should be present and contain back button', function () {
         cy.restoreLoginSession()
         cy.visit("/searchtaxonomy")
+        cy.get('#app').should("exist")
```

Comment:
Braucht man das get('#app').should("exist") an dieser Stelle?

Reply:
Man braucht auf jeden Fall einen Check, dass die Seite geladen wird.
Vorher war das nicht der Fall und es ist hier hin und wieder gefailt, wenn die Seite nicht schnell genug geladen wurde.

Wahrscheinlich würde auch der get() in der nächsten Zeile dafür reichen.
Ich habe get('#app') hier lediglich mit reingenommen, weil ich es in unseren anderen Tests auc hgesehen hatte.

Ist es ok wenn es drin bleibt?

Reply:
ok

Reply:
Finde den Ansatz prinzipiell gut.
Lass es uns dann aber doch einheitlich machen: 
zum beispiel könntest du ein custom `cy.visitAndCheckApp` command machen, dass genau das macht: Es ruft den übergebenen Path auf und überprüft, dass `#app` existiert. 

Und wir benutzen dann in unseren Tests statt cy.visist IMMER cy.visitAndCheckApp. der Check auf App kann dann überall rausfliegen.

Ok?

Reply:
Klingt gut, starte damit jetzt

Reply:
Ich habe das hier getan, aber leider erst nach dem commit gemerkt, dass ich es in den overlay-branch gepusht habe.
Ist das trotzdem ok und kann hier resolvet werden?

Reply:
cherry-pick?

Reply:
cherry-pick durchgeführt